spark-indexer/src/main/java/org/uniprot/store/spark/indexer/main/IndexHPSDocumentsInSolrMain.java

```diff
@@ -26,12 +26,13 @@
 public class IndexHPSDocumentsInSolrMain {
 
     public static void main(String[] args) {
-        if (args == null || args.length != 3) {
+        if (args == null || args.length != 4) {
```

Comment:
I do not think we need an extra parameter here. Because in this step we just save solrDocument to solr. The Taxonomy data is already populated in the document generated in the previous step